Marlin/Configuration.h

```diff
@@ -49,7 +49,7 @@
  *
  * Note: Update also Version.h !
  */
-#define CONFIGURATION_H_VERSION 010100
+#define CONFIGURATION_H_VERSION 010101
```

Comment:
The guidelines for changing the version need to be updated. We'll only be changing it for tagged releases.